### ISSUE TITLE
TD Feature Request - RF Filter for RF Tools

### ISSUE THREAD

**Issue by kennymoto** [CoFH/Feedback#159]
Would it be possible to add a RF filter to TD. The filter would allow RF tools such as a pick axe to be pulled from machinces or inventory based on the RF stored. Specifically I would like to pull a RF Pick from an aut activator when it is below a certain RF level. Thanks!

**Comment by KingLemming** [comment]
Numerically or as a percentage?

Can probably do both actually. Hrm.

**Comment by kennymoto** [comment]
percentage works fine for me. thanks King!

**Comment by evenas98** [comment]
i would like this!

**Comment by KingLemming** [comment]
By the way, NBT can already achieve this, just a heads up.

**Comment by kennymoto** [comment]
Yes with RF tools. The problem comes in with other pick axes from other mods. Tinkers tools use different power levels to break different blocks. Based on hardness? Then you must capture a tool at multiple levels of RF to catch it. It would be helpful to have a filter that handles %. Thanks!

**Comment by Frontrider** [comment]
For tinkers tools a workaround: pull it out if its lost any amount of power. The activator can't do mutch whitout a tool and its gets recharged in seconds, so there shouldn't be any problem.
